packages/components/ContextualMenu/src/ContextualMenuItem.tsx

```diff
@@ -107,6 +107,8 @@ export const ContextualMenuItem = compose<ContextualMenuItemType>({
         onMouseEnter: onMouseEnter,
         onMouseLeave: onMouseLeave,
         accessibilityLabel: accessibilityLabel,
+        focusable: !disabled,
+        accessible: !disabled,
```

Comment:
@lenahong these changes affect win32. Are these OK? I can make them macOS only if not.

Reply:
Nevermind, I just made this macOS specific. But also added more accessbilityprops like AccessibilityState =)